netebpfext/net_ebpf_ext.h

```diff
@@ -161,7 +185,8 @@ net_ebpf_extension_get_callout_id_for_hook(net_ebpf_extension_hook_id_t hook_id)
  * @param[in]  condition_count Count of filter conditions.
  * @param[in]  conditions Common filter conditions to be applied to each filter.
  * @param[in]  raw_context Caller supplied context to be associated with the WFP filter.
```

Comment:
```suggestion
 * @param[in]  filter_context Caller supplied context to be associated with the WFP filter.
```
Also please fix line 184 and 197 to match each other

Reply:
Fixed.